dora/core/common/src/main/java/alluxio/wire/FileInfo.java

```diff
@@ -336,6 +338,22 @@ public Map<String, byte[]> getXAttr() {
     return mXAttr;
   }
 
+  /**
+   * @return the ufs name
+   */
+  @Nullable
+  public String getUfsName() {
+    return mUfsName;
```

Comment:
what's the difference between this and `mUfsPath`?

Reply:
it's actually ufs type. i renamed it